app/models/LocalReferenceNumber.scala

```diff
@@ -34,7 +34,7 @@ object LocalReferenceNumber {
       case _                => None
     }
 
-  implicit def reads: Reads[LocalReferenceNumber] =
+  implicit val defaultReads: Reads[LocalReferenceNumber] =
```

Comment:
Does defaultReads imply there's another reads defined somewhere else?

Reply:
Some hangover from an old solution. Will put it back :)